contentcuration/contentcuration/ricecooker_versions.py

```diff
@@ -10,7 +10,7 @@
     pass
 
 VERSION_OK_MESSAGE = "Ricecooker v{} is up-to-date."
-VERSION_SOFT_WARNING = "0.5.6"
+VERSION_SOFT_WARNING = "0.6.21"
 VERSION_SOFT_WARNING_MESSAGE = "You are using Ricecooker v{}, however v{} is available. You should consider upgrading your Ricecooker."
 VERSION_HARD_WARNING = "0.3.13"
```

Comment:
Not sure if you wanted to update this to 0.6.21 instead? This is the one that will raise a hard error

Reply:
Yes indeed. Fixed in 8b1da51.